docs/configure.md

```diff
@@ -332,17 +319,25 @@ GITSERVERS=$(for i in `seq 0 $(($REPLICA_COUNT-1))`; do echo -n "gitserver-$i.gi
 # Update SRC_GIT_SERVERS environment variable in other services
 find . -name "*yaml" -exec sed -i.sedibak -e "s/value: gitserver-0.gitserver:3178.*/value: $GITSERVERS/g" {} +
 
-IDX_SEARCH=base/indexed-search/indexed-search.PersistentVolumeClaim.yaml
+IDX_SEARCH=base/indexed-search/indexed-search.StatefulSet.yaml
 
-# Update the storage requested in indexed-search's persistent volume claim
-cat $IDX_SEARCH | yj | jq --arg REPLICA_COUNT "$REPLICA_COUNT" '.spec.resources.requests.storage = ((($REPLICA_COUNT |tonumber)  * 200) | tostring)+"Gi"' | jy -o $IDX_SEARCH
+# Update indexed-search replica count
+cat $IDX_SEARCH | yj | jq ".spec.replicas = $REPLICA_COUNT" | jy -o $IDX_SEARCH
 
 # Delete sed's backup files
 find . -name "*.sedibak" -delete
 ```
 
 Commit the outstanding changes.
 
+## Configure indexed-search replica count
+
+Increasing the number of `indexed-search` replicas can improve performance and reliability when your instance contains a large number of repositories. Repository indexes are consistently striped across all `indexed-search` replicas.
```

Comment:
What does striped mean in this context?

Reply:
I am using the same language we use for the gitserver documentation above. Do you think I should change it to consistently hashed or consistently shared?

Reply:
How about: "are distributed evenly"

Reply:
```suggestion
Increasing the number of `indexed-search` replicas can improve performance and reliability when your instance contains a large number of repositories. Repository indexes are distributed evenly across all `indexed-search` replicas.
```